pkg/injector/pod_patch.go

```diff
@@ -629,6 +643,10 @@ func getSidecarContainer(annotations map[string]string, id, daprSidecarImage, im
 
 	HTTPStreamRequestBodyEnabled := HTTPStreamRequestBodyEnabled(annotations)
 
+	if existPlacementAddressesAnnotation(annotations) {
+		placementServiceAddress = getPlacementAddresses(annotations)
+	}
+
```

Comment:
@yaron2 
It will only take effect when this annotation is added.

Reply:
LGTM